src/style.css

```
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: grey;
```

```suggestion
  background-color: #ccc;
```
рекомендую через код

---

Изменил, спасибо